packages/clay-drop-down/src/index.tsx

```diff
@@ -72,11 +79,19 @@ const ClayDropDown: React.FunctionComponent<Props> & {
 	const triggerElementRef = React.useRef<HTMLButtonElement>(null);
 	const menuElementRef = React.useRef<HTMLDivElement>(null);
 
+	const ContainerElement = containerElement;
```

Comment:
Just a Tip, you can define an alias for `containerElement` without having to declare a variable here.

```
containerElement: ContainerElement = 'div',
```